Replicator/tests/ReplicatorAPITest.cc

```diff
@@ -854,8 +854,8 @@ TEST_CASE_METHOD(ReplicatorAPITest, "Stop after transient connect failure", "[C]
     
     waitForStatus(kC4Offline);
     
-    _numCallbacksWithLevel[kC4Connecting] = 0;
-    waitForStatus(kC4Connecting);
+    _numCallbacksWithLevel[kC4Offline] = 0;
+    waitForStatus(kC4Offline);
```

Comment:
There is also waitForStatus(kC4Offline) in line 855. Should one get removed? or Is the test also testing retry (if so, please add a comment there as well) ?